docs/topics/js/js-running-tests.md

```diff
@@ -3,8 +3,7 @@
 The Kotlin Multiplatform Gradle plugin lets you run tests through a variety of test runners that can be specified via the Gradle
 configuration.
 
-When you create a multiplatform project, the Project Wizard automatically adds test dependencies to all the source sets.
-If you created your project without it, you can add the dependencies manually to make test annotations
+When you create a multiplatform project, you can add test dependencies to all the source sets to make test annotations
```

Comment:
It's confusing here, too: "...you can add dependencies to all the source sets to make functionality available for the JavaScript." You can add a dependency to commonTest, and that's what makes JavaScript work, right?